x-pack/test/plugin_api_perf/test_suites/task_manager/index.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export default function({ loadTestFile }: { loadTestFile: (file: string) => void }) {
+  describe.skip('task_manager_perf', function taskManagerSuite() {
```

Comment:
`describe.skip` makes this seem like a flaky test or something that should be enabled. Maybe we just need to add a comment explaining why it's skipped?

Reply:
Yeah, fair, I'll do that

Reply:
If it's `skip` here so it never runs in CI, but allows us to remove the `.skip` to run manually locally (guessing this is how this works), I wonder if we even want to make this CI-able at all.  It would probably be enough to get the task runner bits running in FTS, and then be able to interact with that with `curl` from the command line.  Unless we think we will have a non-skippable runnable CI version of this soon-ish.

Reply:
Yeah don't think we would ever use CI to perform a load test.  Given that, don't think  I would frame this as a functional test.  Probably should move this to a `manual_load_test` dir or some such to make it 100% clear what this is.

Reply:
There's an example Patrick did for manually testing actions: https://github.com/elastic/kibana/tree/master/x-pack/test/alerting_api_integration/security_and_spaces/tests/actions/manual. A start from that + @pmuellr's thoughts could do the trick.

Something could be created alongside the task manager test suite: https://github.com/elastic/kibana/tree/master/x-pack/test/plugin_api_integration/test_suites/task_manager.

Reply:
I'm not sure I'm following the issue.
What's worrying you? That it's under the `test` folder?

I'm not sure I follow what the worry is about it being run like our existing tests- to me that reduced the cognitive load and lets developers run this the same way they'd run other tests, just with one difference: they need to remove the `.skip` in one place.

To me moving this to a different way of wiring it up and having to run it manually using curl seems like more of a pain for people who want to use it later (also, more likely they'll never realise it exists).
No? 🤷‍♂️

Reply:
@mikecote  I had originally had it in the task_manager folder, but moved it out as I felt it was different enough to warrant it's own folder.
Especially if it is't wired in like task_magaer already is.

Do you feel differently?